src/TraceEvent/Stacks/Linux/LinuxPerfScriptStackSource.cs

```diff
@@ -333,6 +343,12 @@ public void AddSamples(IEnumerable<StackSourceSample> _samples)
             SampleEndTime = samples.Last().TimeRelativeMSec;
         }
 
+        public void AddSample(LinuxPerfScriptStackSourceSample sample)
```

Comment:
You'd be able to drop this method with the suggestion above.

Reply:
see above